test/model/src/driver/debug_log_parser.cpp

```diff
@@ -447,6 +484,35 @@ debug_log_parser::apply(kv_transaction_ptr txn, const txn_timestamp &op)
     txn->set_commit_timestamp(op.commit_ts);
 }
 
+/*
+ * debug_log_parser::row_truncate --
+ *     Apply the given operation to the model.
+ */
+void
+debug_log_parser::apply(kv_transaction_ptr txn, const row_truncate &op)
+{
+    /* Handle metadata operations. */
+    if (op.fileid == 0) {
+        throw model_exception("Unsupported metadata operation: row_truncate");
+        return;
```

Comment:
```suggestion
```

Reply:
Yes, that's leftover from what was there previously, thanks for catching that.